destiny/destiny.cache.js

```diff
@@ -55,9 +71,17 @@ class DestinyCache {
      * @returns {Promise}
      */
     async getVendor(vendorHash) {
-        const res = await this.client.get(vendorHash);
+        try {
+            const res = await this.client.get(vendorHash);
+
+            return res ? JSON.parse(res) : undefined;
+        } catch (err) {
+            if (!(err instanceof RedisErrors.RedisError)) throw err;
```

Comment:
I don't want a down Redis Server to result in error responses. If the Redis client is not connected, move on without it.